src/pykeen/evaluation/evaluator.py

```diff
@@ -143,6 +143,8 @@ def evaluate(
         restrict_entities_to: Optional[torch.LongTensor] = None,
         do_time_consuming_checks: bool = True,
         additional_filter_triples: Union[None, MappedTriples, List[MappedTriples]] = None,
+        *,
+        mode: Optional[Mode] = None,
```

Comment:
I would suggest moving `None` into the `Mode` type, since we use it to refer to a certain mode (transductive) which cannot be addressed otherwise.

Reply:
Will dl